recipe/meta.yaml

```diff
@@ -1,26 +1,27 @@
-{% set version = "0.6.7.post3" %}
-{% set lib_version = "0.6.3" %}
+{% set version = "1.0.3" %}
+{% set lib_version = "1.0.0" %}
 
 package:
   name: osqp
   version: {{ version }}
 
 source:
   url: https://github.com/osqp/osqp-python/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 033692f64877e7a8ec6bb1bbef321e2da467e75cf346ef5e48f601dae498a9b9
+  sha256: f50db86014085255230d93419c2cbd2ec468fa4d01aee8f3b441fb2d04322407
   patches:
-    - patches/0001-Disable-MKL-Tests-on-nonx86.patch     # [not x86_64]
-    - patches/0002-use-ninja-universally.patch
-    - patches/0003-point-to-our-own-builds.patch
-    - patches/0004-do-not-rebuild-libosqp-here.patch
-    - patches/0005-disable-codegen-ABI-is-not-compatible.patch
+    - patches/0001-Link-against-shared-osqp-library.patch
+    - patches/0002-Disable-codegen-ABI-is-not-compatible.patch
+    # skip tests that require pytorch
+    - patches/0003-Remove-nn_test-due-to-pytorch.patch  # [ppc64le]
```

Comment:
Removing tests through patches permanently attracts rebase conflicts. It's easier to just skip that module through pytest.

Reply:
I tried this initially, but pytest imports nn_test.py initially, which fails due to missing torch. Alternatively, we can skip tests altogether on ppc64le.

Reply:
Then you could reduce the patch to just removing out the pytorch import (still using the skip obviously)